ts/src/bitfinex.ts

```diff
@@ -1677,7 +1711,7 @@ export default class bitfinex extends Exchange {
     }
 
     nonce () {
-        return this.milliseconds ();
+        return this.milliseconds () * 1000;
```

Comment:
why are we changing this?

Reply:
if I don't change it, I get 

```
% bitfinex fetchBalance
2024-07-09T06:00:44.843Z
Node.js: v18.12.0
CCXT v4.3.56
bitfinex.fetchBalance ()
InvalidNonce bitfinex {"message":"Nonce is too small."}
---------------------------------------------------
[InvalidNonce] bitfinex {"message":"Nonce is too small."}
```

but this is with every method

Reply:
@samgermain We can keep it the bigger the nonce the better was just asking

From the moment you use nano seconds your API keys will never accept milliseconds again.